src/commands/generate.ts

```diff
@@ -0,0 +1,276 @@
+import Command from '../base';
+// eslint-disable-next-line
+// @ts-ignore
+import AsyncAPIGenerator from '@asyncapi/generator';
+import path from 'path';
+import { load, Specification } from '../models/SpecificationFile';
+import { watchFlag } from '../flags';
+import { isLocalTemplate, Watcher } from '../utils/generator';
+import { ValidationError } from '../errors/validation-error';
+
+import type { Example } from '@oclif/core/lib/interfaces';
+
+const red = (text: string) => `\x1b[31m${text}\x1b[0m`;
+const magenta = (text: string) => `\x1b[35m${text}\x1b[0m`;
+const yellow = (text: string) => `\x1b[33m${text}\x1b[0m`;
+const green = (text: string) => `\x1b[32m${text}\x1b[0m`;
+
+interface ParsedFlags {
+  params: Record<string, string>,
+  disableHooks: Record<string, string>,
+  mapBaseUrlToFolder: {
+    url: string,
+    folder: string
+  }
+}
+
+export default class Generate extends Command {
+  static description = 'Generator is a tool that you can use to generate whatever you want basing on the AsyncAPI specification file as an input.';
+
+  static examples: Example[] = [
+    'asyncapi generate asyncapi.yaml @asyncapi/html-template'
+  ];
+
+  static flags = {
+    help: Flags.help({ char: 'h' }),
+    'disable-hook': Flags.string({
+      char: 'd',
+      description: 'disable a specific hook type or hooks from a given hook type',
+      multiple: true
+    }),
+    install: Flags.boolean({
+      char: 'i',
+      default: false,
+      description: 'installs the template and its dependencies (defaults to false)'
+    }),
+    debug: Flags.boolean({
+      description: 'enable more specific errors in the console'
+    }),
+    'no-overwrite': Flags.string({
+      char: 'n',
+      multiple: true,
+      description: 'glob or path of the file(s) to skip when regenerating'
+    }),
+    output: Flags.string({
+      char: 'o',
+      description: 'directory where to put the generated files (defaults to current directory)',
+      required: true
+    }),
+    'force-write': Flags.boolean({
+      default: false,
+      description: 'force writing of the generated files to given directory even if it is a git repo with unstaged files or not empty dir (defaults to false)'
+    }),
+    'watch-template': watchFlag(
+      'watches the template directory and the AsyncAPI document, and re-generate the files when changes occur. Ignores the output directory.'
+    ),
+    param: Flags.string({
+      char: 'p',
+      description: 'additional param to pass to templates',
+      multiple: true
+    }),
+    'map-base-url': Flags.string({
+      description: ' maps all schema references from base url to local folder'
+    })
+  }
+
+  static args = [
+    { name: 'asyncapi', description: 'Local path or url pointing to AsyncAPI specification file', required: true },
+    { name: 'template', description: 'Name of the generator template like for example @asyncapi/html-template or https://github.com/asyncapi/html-template', required: true }
```

Comment:
```suggestion
    { name: 'asyncapi', description: '- Local path or url pointing to AsyncAPI file', required: true },
    { name: 'template', description: '- Name of the generator template like for example @asyncapi/html-template or https://github.com/asyncapi/html-template', required: true }
```

this way it reads better in the terminal